packages/edit-site/src/components/editor/style.scss

```diff
@@ -15,11 +15,20 @@
 }
 
 // Adjust the position of the notices
-.edit-site .components-editor-notices__snackbar {
-	position: fixed;
-	right: 0;
-	bottom: 40px;
-	padding-left: 16px;
-	padding-right: 16px;
+.edit-site {
+	.components-editor-notices__snackbar {
+		position: fixed;
+		right: 0;
+		bottom: 40px;
+		padding-left: 16px;
+		padding-right: 16px;
+	}
+
+	@media (min-width: $break-medium) {
+		.is-inserter-opened .components-editor-notices__snackbar,
+		.is-list-view-opened .components-editor-notices__snackbar {
+			padding-left: calc(16px + #{$nav-sidebar-width});
```

Comment:
All of the `16px` values in this PR (I count 4) probably should be `$grid-unit-20`.